srcpkgs/arc-theme/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'arc-theme'
 pkgname=arc-theme
-version=20211018
+version=20220223
 revision=1
 build_style=meson
 hostmakedepends="glib-devel pkg-config gnome-shell sassc inkscape cinnamon"
 makedepends="gtk+3-devel gtk4-devel"
 depends="gnome-themes-extra gtk-engine-murrine"
 short_desc="Flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"
```

Comment:
because it has gtk4-devel in makedepends, I don't think it's just for gtk3 and gtk2, am I right?
```suggestion
short_desc="Flat theme with transparent elements for GTK and GNOME Shell"
```

Reply:
sure, should be fine